+llvm.org fails in linux

Downloading the tarball for llvm fails part way through on the first run. After the cache is corrupted so reattempts fail.

First run (no cache):
`sh <(curl https://tea.xyz) +llvm.org sh`
```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13759  100 13759    0     0  93598      0 --:--:-- --:--:-- --:--:-- 94239
llvm.org=16.0.1 downloading 977 MiB 44%
Killed
```

Second run (cache hit):
`sh <(curl https://tea.xyz) +llvm.org sh`
```
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 13759  100 13759    0     0   104k      0 --:--:-- --:--:-- --:--:--  104k
llvm.org=16.0.1 cache: hit

gzip: stdin: unexpected end of file
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now

panic: spilt tea. we’re sorry and we’ll fix it… but you have to report the bug!

    https://github.com/teaxyz/cli/issues/new?title=panic%3Atar+exited+with+stat
us+2                                                                           
----------------------------------------------------->> attachment begin
Error: tar exited with status 2
    at install (file:///tmp/01883f3a/src/prefab/install.ts:96:19)
    at async install (file:///tmp/01883f3a/src/hooks/useExec.ts:117:25)
    at async default (file:///tmp/01883f3a/src/hooks/useExec.ts:79:38)
    at async run (file:///tmp/01883f3a/src/app.main.ts:32:63)
    at async file:///tmp/01883f3a/src/app.ts:11:5
<<----------------------------------------------------- attachment end
```
Other versions also fail:

- 12.0.1 
  ```
  llvm.org=12.0.1 downloading 589 MiB 70%
  Killed
  ```
- 13.0.1 
  ```
  llvm.org=13.0.1 downloading 712 MiB 57%
  Killed
  ```
- 13.0.1 
  ```
  llvm.org=13.0.1 downloading 712 MiB 57%
  Killed
  ```
- 14.0.6 
  ```
  llvm.org=14.0.6 downloading 796 MiB 45%
  Killed
  ```
- 15.0.6 
  ```
  llvm.org=15.0.6 downloading 862 MiB 41%
  Killed
  ```

I don't see any pattern in the download size or percentage, but it consistently gets cut off part way though.

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -1,0 +1,5 @@
+{
+  "hostRequirements": {
+    "memory": "8gb"
+  }
+}

